### PR TITLE
✏️ Improve the speaking resources page for post-CFP stuff

### LINKED ISSUE #6: 2023: Revamp speaker resources
- Simplify slide guidelines 
- Add talk/script guidelines 
- Call out using gender-neutral language 
- Call out being respectful, even when offering critique

### PR DESCRIPTION
**Pull request by drewbrew**
1. Add notes about accessibility
2. Refactor slide guidelines to be separate from talk guidelines
3. Remove some notes about which software to use
4. Add the ability to submit talk slides in advance
5. Give some quick tips for hooking your laptop up to the projector

Fixes #6.

Screenshots:
image

image

image

image

### CODE REVIEW

File: _pages/speaking-resources.md [hunk 1]
```diff
@@ -26,21 +28,42 @@ Presenters, regardless of experience, sometimes want a little help. If you’d l
...
+### Accessibility
+
+* You can submit a draft copy of your slides with notes indicating what you plan to say a minimum of three weeks before your talk to give the captioners time to train on your data and therefore improve caption accuracy. This is extremely helpful to our attendees who read the live captions if you're using jargon beyond "Django" and "the ORM".
+* These slides will not be shared outside of the organizing team and the captioners. Send the slides to hello AT djangocon DOT us when you are ready. It's perfectly fine if your slides and notes change between the draft and final version.
+* When describing media in your talk (images, screenshots, graphs, videos, etc.), please give a verbal description of what you're showing so that blind or vision-impaired attendees can understand what you're describing.
+  * Example: instead of "here's a graph showing performance improvements with our new async code," say "here's a line graph showing our average response time that decreased from 450 milliseconds to 150 milliseconds after adding our new async code."
+
+### General
+
+* Use gender-neutral language wherever possible. Don't default to using "he" or "him" everywhere.
```
Comment (tim-schilling) [inline _pages/speaking-resources.md:42]: Do we want to call out "Hey guys" (https://heyguys.cc/) here?

File: _pages/speaking-resources.md [hunk 2]
```diff
@@ -26,21 +28,42 @@ Presenters, regardless of experience, sometimes want a little help. If you’d l
...
+* Be respectful of other speakers, your audience, and the community at large, especially when offering critiques. It's possible to say "this needs improvement" without saying "this is bad."
+* Consider including your social media handle(s) on your opening and closing slides! If you have space, you can even include them on the footer of every slide.
+* Consider publishing your slides after your talk (on the platform of your choice) and sharing the link with the #djangocon hashtag on social media.
```
Comment (tim-schilling) [inline _pages/speaking-resources.md:45]: Can we use camel casing on the hashtags? It's preferred on Mastodon.

File: _pages/speaking-resources.md [hunk 3]
```diff
@@ -26,21 +28,42 @@ Presenters, regardless of experience, sometimes want a little help. If you’d l
...
+### Format
+
 * Minimal slides are best—-avoid walls of text and long lists of bullets.
-* Aim for high contrast slides, avoiding colors that may be difficult to see for those with colorblindness. (You can [check your contrast online](http://webaim.org/resources/contrastchecker/); you just need the hex codes for your colors!)
 * Light background with dark text is easiest to read; be mindful that the projection screen is white.
-* Make text as large as possible, at least 68pt.
-* Choose fonts with adequate spacing between letters, and avoid thin or cursive fonts.
-* Leave the bottom ⅓ of your slides free of text to ensure nothing is obscured.
+ Choose fonts with adequate spacing between letters, and avoid thin or cursive fonts.
+* Leave the bottom 10⅓ of your slides free of text to ensure nothing is obscured by other attendees' heads.
```
Comment (tim-schilling) [inline _pages/speaking-resources.md:58]: > 10⅓

Did you mean ⅓?

File: _pages/speaking-resources.md (hunk 2 above)
Comment (drewbrew) [reply _pages/speaking-resources.md:45]: Definitely

File: _pages/speaking-resources.md (hunk 3 above)
Comment (drewbrew) [reply _pages/speaking-resources.md:58]: Yep, that was me failing at reading VS Code's font. I thought that was a percent sign.

Comment (drewbrew) [reply _pages/speaking-resources.md]: I changed it to 25% to make it a little less onerous.

**Review by tim-schilling — APPROVED** [review]

**Pull request merged**